cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -6,6 +6,7 @@
 import pytest
 
 from cfme import test_requirements
+from cfme.fixtures.soft_assert import soft_assert
```

Comment:
This shouldn't be directly imported. It's registered automagically, and should just be included in the fixture list at the function, class, or module level.